packages/preview/cnam-template/1.0.0/typst.toml

```diff
@@ -0,0 +1,28 @@
+# for a description of available keys, see https://github.com/typst/packages/?tab=readme-ov-file#package-format
+
+[package]
+name = "cnam-template"
```

Comment:
If this should be an official template, a faculty staff member needs to send an email to hello@typst.app endorsing this package. Otherwise the name needs to be prefixed by some non-descriptive part. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules)

Reply:
That would work :+1: